### DISCUSSION TITLE
Can anyone test something for me?

### DISCUSSION THREAD

**Discussion by moraroy** [moraroy/NonSteamLaunchers-On-Steam-Deck#69] [Q&A] [answered]
Once the steam shortcuts are made into your steam library, can you test if you can delete the steam shortcut and if so does Steam delete the prefix folder? This new update people are saying Steam... deletes the prefix now.

**Comment by daviator** [comment]
I tried it with GOG Launcher and the prefix folder and contents was left untouched. I used the "remove non-Steam game from your library" option. Is that the one that should be tested?

**Reply by moraroy** [reply]
Perfect thank you! Yes it seems the new update doesn't effect NonSteamLaunchers. Whether that's good or bad remains to be seen. I don't know I feel about removing the shortcut and then all of the games and prefixes being deleted , and shader cache. So I'd say it's good that it doesn't delete. The script comes with "Start Fresh" button that deletes all that. Although maybe I will add in shader cache in that button as well so it deletes it too. What do you think?@daviator

**Comment by daviator** [comment, marked as answer]
The setup right now makes sense to me. When I want to uninstall specific launcher(s) I can use your script. Right now I have to remove the steam shortcut myself (no problem), maybe the uninstall button can also remove the steam shortcut?
I used the "Start Fresh" button yesterday and everything was cleaned (except the shortcuts in the steam library, still no problem, but fyi).
So imho everything works well! Thanks for the awesome script!

**Reply by moraroy** [reply]
@daviator hey brother I hate to bother ya, but could you also test the new .sh file I put out yesterday, I'm trying to get rod of using "setup tools" and just using the vdf library instead straight from the repository. Basically just see if the shortcuts still add to your library.

**Reply by daviator** [reply]
Sorry it was very busy in my life and around me. I tested the version you released yesterday and it works!
